docs/src/main/sphinx/release/release-465.md

```diff
@@ -0,0 +1,71 @@
+# Release 465 (15 Nov 2024)
+
+## General
+
+* Add the {func}`cosine_similarity` function for dense vectors. ({issue}`23964`)
+* Add support for reading geometries in [EWKB
+  format](https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry)
+  with the {func}`ST_GeomFromBinary` function. ({issue}`23824`)
+* Add support for the `ORDER BY` clause in a windowed aggregate function. ({issue}`23929`)
+* Improve performance of queries with selective joins on large build sides. ({issue}`22824`)
+* Fix potential configuration leakage between catalogs using JDBC-based
+  connectors. ({issue}`24058`)
+* Reduce chance of out of memory query failure when `retry-policy` is set to
+  `task`. ({issue}`24114`)
+
+## CLI
+
+* Fix incorrect quoting of output values when the `CSV_UNQUOTED` or the 
+  `CSV_HEADER_UNQUOTED` format is used. ({issue}`24113`)
+
+## BigQuery connector 
+
+* Fix failure when reading views with `timestamp` columns. ({issue}`24004`)
+
+## Cassandra connector
+
+* {{breaking}} Require setting `cassandra.security=PASSWORD` along with
+  `cassandra.username` and `cassandra.password` for  password-based
+  authentication. ({issue}`23899`)
+
+## Clickhouse connector
+
+* Fix insert for tables with the Clickhouse `TimestampWithTimeZone` type. ({issue}`23785`)
+
+## Delta Lake connector
+
+* Fix incorrect results when filtering partition columns on checkpoints with
```

Comment:
I don't understand what this means.

Reply:
The bug shows example queries. I think the column_mapping_mode='NAME' is a Delta Lake specific thing and this fixes a related issue. Not sure how to best simplify .. any input @ebyhr or @findinpath ? Should we be explicit with `column_mapping_mode='NAME'` in the entry?

Reply:
I'm less concerned about the column mapping mode bit than what "filtering partition columns on checkpoints" means.

Reply:
I reworded.. hopefully this is fine now.